recipes/configs/gemma2/27B_qlora_single_device.yaml

```
@@ -0,0 +1,115 @@
+# Config for multi-device QLoRA finetuning in lora_finetune_single_device.py
+# using a gemma2 27B model
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download google/gemma-2-27b --ignore-patterns "gemma-2-27b.gguf"  --hf-token <HF_TOKEN>
+#
+# To launch on a single device, run the following command from root:
+#   tune run lora_finetune_single_device --config gemma2/27B_qlora_single_device
+#
+# You can add specific overrides through the command line. For example
+# to override the checkpointer directory while launching training
+# you can run:
+#   tune run lora_finetune_single_device --config gemma2/27B_qlora_single_device checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#
+# This config works only for training on single device.
+
+# Tokenizer
+tokenizer:
+  _component_: torchtune.models.gemma.gemma_tokenizer
+  path: /tmp/gemma-2-27b/tokenizer.model
+
+# Dataset
+dataset:
+  packed: False # Set to true for great speed ups
+  _component_: torchtune.datasets.alpaca_dataset
+seed: null
+shuffle: True
+
+# Model Arguments
+model:
+  _component_: torchtune.models.gemma2.qlora_gemma_27b
+  lora_attn_modules: ['q_proj', 'k_proj', 'v_proj']
+  apply_lora_to_mlp: True
+  lora_rank: 64
+  lora_alpha: 128
+  lora_dropout: 0.0
+
+checkpointer:
+  _component_: torchtune.training.FullModelHFCheckpointer
+  checkpoint_dir: /tmp/gemma-2-27b/
+  checkpoint_files:
+    filename_format: model-{}-of-{}.safetensors
+    max_filename: 00024
```

I think all usages of max_filename should look like this instead
```suggestion
    max_filename: "00024"
```